packages/appium/docs/en/ecosystem/tools.md

```diff
@@ -18,14 +18,21 @@ Find downloads and more information on its GitHub page: [Appium Inspector](https
 
 ### Appium Doctor
 
-The `appium-doctor` CLI tool can be used to validate proper environment setup for Android and iOS
-automation, and identify any common configuration issues. It can be installed with `npm`:
+Each driver has its own `doctor` command as of Appium 2.4.0.
+If a driver/plugin has its own requirement, the `doctor` command will help to setup the requirement.
```

Comment:
We should still keep information on what the doctor command actually does. I would suggest something like this:

```
Appium Doctor is a command-line tool built into all officially supported Appium drivers and plugins. The command can be used to validate if a driver or plugin has all of its prerequisites and other environment details set up correctly.

More information on this command can be found in the [Command-Line Usage documentation](../cli/extensions.md#doctor).
```

Specifying the minimum Appium version is not required, since this change will only appear in the docs for Appium >=2.6. Examples can also be omitted, since the CLI docs page already has them.

Reply:
oh, yea. I thought the versioning was removed as:

![Screenshot 2024-05-19 at 12 02 03 PM](https://github.com/appium/appium/assets/5511591/25179e71-ccd6-459e-a0e1-8e4dc0dee082)

but other pages have:

![Screenshot 2024-05-19 at 12 02 07 PM](https://github.com/appium/appium/assets/5511591/1d5f39f2-1bfa-41df-909e-b3edad8fb96c)

Reply:
It's only the welcome page that doesn't have the version, since that page is slightly customised.

Reply:
Updated